### ISSUE TITLE
hi how add your plugin to get through proxy 

### ISSUE THREAD

**Issue by mohammadhoseinazad** [artemave/translate_onhover#77]
hi how to add your plugin to get through proxy 
what site do you use for translation to add it as well

**Comment by artemave** [comment]
`https://translate.googleapis.com`

See https://github.com/artemave/translate_onhover/blob/master/manifest.json#L33

**Comment by mohammadhoseinazad** [comment]
I saw this line in js and add it but it does not perform correctly after a while with connected proxy

**Comment by artemave** [comment]
Yeah, that is probably Google translate API rate limiting you. There is not work around for that at the moment unfortunately.

**Comment by mohammadhoseinazad** [comment]
So how can i change it to (https://www.vocabulary.com/dictionary/$hi$) for
instance  that can look up for hi with this url

**Comment by artemave** [comment]
This isn't trivial. You'd need to hook into their API (if there is one) and adjust to their data format.

**Comment by srsh-kh** [comment]
also for me this extension doesn't work and I have to clear all cache (firefox) and then it works. I use vpn (softether)
